### PR TITLE
spring cleaning and first hop

### PR DESCRIPTION
**Pull request by TSKsmiley**

### CODE REVIEW

**Comment by NybyDK** [comment]
I DONT GIVE A FUCK ABOUT THE FUCKING CODE! i just want to download this stupid fucking application and use it https://github.com/sherlock-project/sherlock#installation
 
WHY IS THERE CODE??? MAKE A FUCKING .EXE FILE AND GIVE IT TO ME. these dumbfucks think that everyone is a developer and understands code. well i am not and i don't understand it. I only know to download and install applications. SO WHY THE FUCK IS THERE CODE? make an EXE file and give it to me. STUPID FUCKING SMELLY NERDS 🤓

**Review by NybyDK — CHANGES_REQUESTED** [review]
anti merge, we need #108

File: src/lib/components/Packet.svelte [hunk 1]
```diff
@@ -59,6 +74,6 @@
 
 <style>
   circle {
-    fill: red;
+    fill: #6495ed;
```
Comment (JakobTopholt) [inline src/lib/components/Packet.svelte:77]:
```suggestion
    fill: #6495ed; /*light blue*/
```

**Review by JakobTopholt — CHANGES_REQUESTED** [review]
I generally speaking can approve it with our without my addition, but I'd like my addition

**Review by JakobTopholt — APPROVED** [review]

**Review by NybyDK — APPROVED** [review]

**Review by JakobTopholt — APPROVED** [review]

**Pull request merged**